scrapers/distgit.py

```diff
@@ -199,11 +184,9 @@ def get_distgit_data(self, since, until):
         """
         sql = """\
             SELECT c.commit_id, c.author, c.author_date, c.committer, c.commit_date, c.log_message,
-                c.sha, bz.bugzilla_id, bz.type as bugzilla_type, p.push_date, p.push_ip, p.module,
-                p.ref, p.push_id, p.pusher
+                c.sha, bz.bugzilla_id, bz.type as bugzilla_type, p.module,
+                p.ref
             FROM gitbz.git_commits as c
-            LEFT JOIN gitbz.git_push_commit_map as map ON c.commit_id = map.commit_id
-            LEFT JOIN gitbz.git_pushes as p ON p.push_id = map.push_id
```

Comment:
If you remove this `LEFT JOIN`, you would not be able to retrieve p.module and p.ref. Hence, we would probably need this statement